packages/firestore/exp/src/api/database.ts

```diff
@@ -479,3 +489,41 @@ function verifyNotInitialized(firestore: FirebaseFirestore): void {
     );
   }
 }
+
+export function _loadBundle(
+  firestore: FirebaseFirestore,
+  bundleData: ArrayBuffer | ReadableStream<Uint8Array> | string
+): LoadBundleTask {
+  firestore._verifyNotTerminated();
+
+  const resultTask = new LoadBundleTask();
+
+  firestore._queue.enqueueAndForget(async () => {
+    const databaseId = (await firestore._getConfiguration()).databaseInfo
+      .databaseId;
+    const reader = createBundleReader(bundleData, newSerializer(databaseId));
+    const syncEngine = await getSyncEngine(firestore);
+
+    loadBundleSyncEngine(syncEngine, reader, resultTask);
```

Comment:
FWIW, I have lately followed Gil's suggestions more strictly and prefixed the free standing APIs with the class name.

That would mean s/loadBundleSyncEngine/syncEngineLoadBundle/

Reply:
Done.